docs/README.md

```diff
@@ -84,19 +90,179 @@ slug: /migrations/v5-to-v6 # the url of the file
 ---
 ```
 
+The `link` key in `_category_.json` determines if the category has an introductory page that comes before any content pages. If `link` is `null`, then the category does not have an introductory page. If there is a markdown file you wish to link, you should do
+
+```json
+{
+  "label": "Sidebar Label",
+  "position": 1, // position of the category in the sidebar
+  "link": { "type": "doc", "id": "intro" }
+}
+```
+
+The `id` key can be defined in the frontmatter of the markdown file. Or, you can use the id tag as an extension to the url of the current page. For example, the following frontmatter on a markdown file in the same directory as the `_category_.json` file shown above will link to the markdown file:
+
+```yaml
+---
+title: Title
+sidebar_label: Sidebar Label
+sidebar_position: 0 # should be zero for intro pages
+slug: /ibc/upgrades/intro
+---
+```
+
+## Best practices
+
+- Check the spelling and grammar, even if you have to copy and paste from an external source.
+- Use simple sentences. Easy-to-read sentences mean the reader can quickly use the guidance you share.
+- Try to express your thoughts in a concise and clean way.
+- Either Leave a space or use a `-` between the acronyms ADR and ICS and the corresponding number (e.g. ADR 008 or ADR-008, and ICS 27 or ICS-27).
+- Don't overuse `code` format when writing in plain English.
+- Follow Google developer documentation [style guide](https://developers.google.com/style).
+- Check the meaning of words in Microsoft's [A-Z word list and term collections](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms) (use the search input!).
+- We recommend using RFC keywords in user documentation (lowercase). The RFC keywords are: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL. They are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+- Lint the markdown files for documentation with [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli). Run `make docs-lint` (you will need to have `markdownlint-cli` installed, so please follow the [installation instructions](https://github.com/igorshubovych/markdownlint-cli#installation)).
```

Comment:
I guess the reference to `markdownlint-cli` here will be updated with [this other PR](https://github.com/cosmos/ibc-go/pull/3805), right?